build.gradle

```diff
@@ -166,54 +168,64 @@ tasks.register('integrationTest', Test) {
 }
 
 dependencies {
-    api platform("com.exactpro.th2:bom:3.2.0")
-    api ("com.exactpro.th2:cradle-core:${cradleVersion}")
+    api platform("com.exactpro.th2:bom:4.0.0-vulnerabilities-2919290091-SNAPSHOT")
+    api ("com.exactpro.th2:cradle-core:${cradleVersion}") {
+        exclude group: 'org.slf4j', module: 'slf4j-log4j12' // because of the vulnerability
+    }
     api 'com.exactpro.th2:grpc-common:3.11.1'
 
     implementation 'com.google.protobuf:protobuf-java-util'
     implementation 'com.exactpro.th2:grpc-service-generator:3.2.2'
-    implementation ("com.exactpro.th2:cradle-cassandra:${cradleVersion}")
+    implementation ("com.exactpro.th2:cradle-cassandra:${cradleVersion}") {
+        exclude group: 'org.slf4j', module: 'slf4j-log4j12' // because of the vulnerability
+    }
 
     //FIXME: Add these dependencies as api to grpc-... artifacts
     implementation "io.grpc:grpc-protobuf"
     implementation "io.grpc:grpc-core"
     implementation "io.grpc:grpc-netty"
-    implementation "io.grpc:grpc-services:1.32.1" //FIXME: add to bom
+    implementation "io.grpc:grpc-services"
 
     implementation "com.rabbitmq:amqp-client"
 
     implementation "org.jetbrains:annotations"
 
     implementation "org.apache.commons:commons-lang3"
-    implementation "org.apache.commons:commons-collections4:4.4"
+    api("org.apache.commons:commons-collections4:4.4") {
+        because('multimap in MqRouterFilterConfiguration')
+    }
     implementation "org.apache.commons:commons-text:1.9"
     implementation "commons-cli:commons-cli:1.5.0"
 
     implementation "com.fasterxml.jackson.core:jackson-core"
-    implementation "com.fasterxml.jackson.core:jackson-databind"
-    implementation "com.fasterxml.jackson.core:jackson-annotations"
+    api("com.fasterxml.jackson.core:jackson-databind") {
+        because('provide ability to use object mapper in components')
+    }
+    api("com.fasterxml.jackson.core:jackson-annotations") {
+        because('providee ability to use jackson annotations in components')
+    }
     implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr310"
     implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml"
-    // TODO: need to update jackson kotlin module to get rid of kotlin-reflect 1.3.72
     implementation "com.fasterxml.jackson.module:jackson-module-kotlin"
 
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${kotlin_version}"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
 
     implementation 'com.fasterxml.uuid:java-uuid-generator:4.0.1'
 
     implementation 'io.github.microutils:kotlin-logging:2.1.21'
```

Comment:
It is commonly used dependency. Probably, need to be moved to bom or declared as `api` dependecy